### PR TITLE
[Redesign] Address various points of feedback from the community

### PR DESCRIPTION
**Pull request by joelverhagen**
https://github.com/NuGet/NuGetGallery/issues/4432
https://github.com/NuGet/NuGetGallery/issues/4419
https://github.com/NuGet/NuGetGallery/issues/4440
https://github.com/NuGet/NuGetGallery/issues/4441
https://github.com/NuGet/NuGetGallery/issues/4443

Observe the bugs and each commit separately.

### CODE REVIEW

**Comment by joelverhagen** [comment]
🔔

File: src/Bootstrap/less/theme/common-list-packages.less [hunk 1]
```diff
@@ -12,6 +12,8 @@
         font-size: @font-size-h2;
         font-weight: 300;
         line-height: 0.9;
+        word-break: break-all;
```
Comment (scottbommarito) [inline src/Bootstrap/less/theme/common-list-packages.less]: why do we have the same property twice, set to different values?

Comment (joelverhagen) [inline src/Bootstrap/less/theme/common-list-packages.less]: `break-all` is supported on more browsers than `break-word`.

Comment (joelverhagen) [inline src/Bootstrap/less/theme/common-list-packages.less]: `break-word` will override `break-all` if the browser understands it.

**Pull request merged**